server/index.js

```diff
@@ -21,12 +21,15 @@ app.get('/app/:gameid', (req, res) => {
 
 app.get('/api/description/:gameid', async (req, res) => {
   const { gameid } = req.params;
-  if (gameid < 1 || gameid > 100) {
-    res.status(400).json({ error: 'Invalid game id' });
-    return;
-  }
+  // if (gameid < 1 || gameid > 100) {
```

Comment:
Similar to above; it's a small validation, but not necessarily a useless one if it's upped to the right number.